packages/core/src/routes/admin-user.ts

```diff
@@ -0,0 +1,43 @@
+import { userInfoSelectFields } from '@logto/schemas';
+import pick from 'lodash.pick';
+import { InvalidInputError } from 'slonik';
+import { object, string } from 'zod';
+
+import koaGuard from '@/middleware/koa-guard';
+import { findRolesByRoleName } from '@/queries/roles';
+import { findUserById, updateUserById } from '@/queries/user';
+
+import { AuthedRouter } from './types';
+
+export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
```

Comment:
will it be better to just use `/users` route for admin, and `/user` route for current user?

Reply:
make sense

Reply:
cc: @xiaoyijun  as the current user management project is under your supervise.